packages/airbnb-base/README.md

```diff
@@ -168,64 +168,26 @@ module.exports = {
   use: [
     ['@neutrinojs/airbnb-base', {
       eslint: {
-        rules: {
-          'babel/semi': 'off'
+        // For supported options, see:
+        // https://github.com/webpack-contrib/eslint-loader#options
+        // https://eslint.org/docs/developer-guide/nodejs-api#cliengine
+        // The options under `baseConfig` correspond to those
+        // that can be used in an `.eslintrc.*` file.
+        baseConfig: {
+          extends: [
+            'my-custom-config'
+          ],
+          rules: {
+            'babel/semi': 'off'
+          }
         }
       }
     }]
   ]
 };
 ```
 
-## Customizing
-
-To override the build configuration, start with the documentation on [customization](https://neutrinojs.org/customization/).
-`@neutrinojs/airbnb-base` creates some conventions to make overriding the configuration easier once you are ready to
-make changes.
-
-### Rules
-
-The following is a list of rules and their identifiers which can be overridden:
-
-| Name | Description | NODE_ENV |
-| --- | --- | --- |
-| `lint` | Lints JS and JSX files from the `src` directory using ESLint. Contains a single loader named `eslint`. This is inherited from `@neutrinojs/eslint`. | all |
-
-### Information
-
-If you want your preset or middleware to also extend from another **ESLint configuration or preset** that you have made
```

Comment:
This is now covered by all of the defaults above being listed under `baseConfig` instead